model/model.go

```diff
@@ -36,9 +36,10 @@ const (
 type PlayerColor int8
 
 const (
-	Green PlayerColor = iota
-	Blue
-	Red
+	UnsetColor PlayerColor = 0
+	Green      PlayerColor = 1
+	Blue       PlayerColor = 2
+	Red        PlayerColor = 3
```

Comment:
Values that are perissted should always be defined explicitly and should not rely on iota.